test/start-stop.node.js

```diff
@@ -141,7 +141,7 @@ tests.forEach((fOpts) => {
           ipfsd.start(['--should-not-exist'], (err) => {
             expect(err).to.exist()
             expect(err.message)
-              .to.match(/Unrecognized option 'should-not-exist'/)
+              .to.match(/unknown option "should-not-exist"\n/)
```

Comment:
@Stebalien This is a breaking change for the CLI. Not a deal breaker.

Reply:
*sigh*... Yeah, we should have predefined error *codes* but currently we just spit back messages (and occupationally change them).